src/index.js

```diff
@@ -22,6 +25,15 @@ class RealmAPI {
     const data = await this.rest.get('/worlds')
     return data.servers.map(realm => new Realm(this, realm))
   }
+
+  async getRealmBackups (realmId, slotId) {
+    const data = await this.rest.get(`/worlds/${realmId}/backups`)
+    return data.backups.map(e => new Backup(this, { realmId, slotId }, e))
+  }
+
+  async restoreRealmFromBackup (realmId, backupId) {
+    return await this.rest.put(`/worlds/${realmId}/backups?backupId=${backupId}&clientSupportsRetries`, { retryCount: 5 })
```

Comment:
if backupId can contain arbitrary characters, then this call is unsafe. You need to use encodeURIComponent.